docs/pages/concepts/Catalog/catalog.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Catalog
+permalink: concepts/catalog
+redirect_from: catalog
+type: concepts
+abstract: Getting to know about catalog
+language: en
+list: include
+---
+
+# Meshery Catalog
+
+[Meshery Catalog](https://meshery.io/catalog) functions much like a cloud marketplace, providing a user-friendly interface for browsing, discovering, and sharing configurations and patterns for cloud native infrastructure. With Meshery Catalog, you can easily find and deploy Kubernetes-based infrastructure and tools, making it easy to streamline your cloud native development and operations processes.
+
+### Simplify Your Cloud Native Infrastructure Deployment and Management
+
+Meshery Catalog functions much like a cloud marketplace, providing a user-friendly interface for browsing, discovering, and sharing configurations and patterns for cloud native infrastructure. With Meshery Catalog, you can easily find and deploy Kubernetes-based infrastructure and tools, making it easy to streamline your cloud native development and operations processes.
+
+It also supports a collaborative environment, where DevOps engineers can share their experiences, feedback, and best practices with others in the community. Import cloud native patterns published by others into your Meshery Server. Benefit from and build upon each pattern by incorporating your own tips and tricks, then publish and share with the community at-large. This facilitates knowledge-sharing and helps to build a strong ecosystem of cloud native infrastructure experts.
+
+## Add your content to the catalog
+
+To add your content to the Cloud Native Catalog -
+
+1. Head over [meshery](https://github.com/meshery/meshery.io/blob/master/README.md#add-your-content-to-the-cloud-native-catalog) repository and set up your local enviroment.
+1. Create a new file in the catalog collection, under the respective type.
+2. Follow the template in the [_defaults.md](https://github.com/meshery/meshery.io/blob/master/collections/_catalog/_defaults.md) and fill out the details.
+3. Add your pattern file to the [catalog](https://github.com/meshery/meshery.io/tree/master/catalog) folder so that others can download the file.
```

Comment:
This is no longer the process. Please see the instructions on meshery.io/catalog, @yash37158

Reply:
Thanks for your response @lee, i definitely take that into consideration